e2e/playwright/point-click.spec.ts

```diff
@@ -2270,8 +2270,8 @@ chamfer04 = chamfer(extrude001, length = 5, tags = [getOppositeEdge(seg02)])
       cmdBar,
     }) => {
       const initialCode = `sketch001 = startSketchOn('XZ')
-    |> circle(center = [0, 0], radius = 30)
-    extrude001 = extrude(sketch001, length = 30)
+  |> circle(center = [0, 0], radius = 30)
+extrude001 = extrude(sketch001, length = 30)
```

Comment:
This bad formatting of the start code was was causing the edit step failure. Something else is still going on 🆘

Reply:
Yeah that's my view on it